jormungandr/src/stats_counter.rs

```diff
@@ -12,6 +12,7 @@ struct StatsCounterImpl {
     tx_recv_cnt: AtomicUsize,
     block_recv_cnt: AtomicUsize,
     start_time: Instant,
+    slot_start_time: RwLock<Option<SystemTime>>,
```

Comment:
RwLock for this seems totally overkill

Reply:
I couldn't cram SystemTime into a single Atomic (it takes 128 bits) and the data in this structure must be both Sync and internally-mutable :(

Reply:
I would put it in 2 atomic u64. it really doesn't matter is someone grab this value in the middle of an update, and not totally sure we need to keep the nanoseconds precision anyway

Reply:
In jormungandr there is `SecondsSinceUnixEpoch` which is a `u64` already. Does not contain the nanoseconds precision. And it provides compatibility back into/from `SystemTime`.